airgun/views/contentview.py

```diff
@@ -58,6 +62,50 @@ def is_displayed(self):
         )
 
 
+class ContentViewCopyView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    new_name = TextInput(id='copy_name')
+    create = Text(".//button[@type='submit']")
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Content Views'
+                and len(self.breadcrumb.locations) == 3
+                and self.breadcrumb.read() == 'Copy'
+        )
+
+
+class ContentViewRemoveView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    conflicts = Text("//div[@ng-show='conflictingVersions.length > 0']")
+    table = SatTable('.//table')
+    remove = Text(".//button[@ng-click='delete()']")
+    cancel = Text(
+        ".//a[contains(@class, 'btn')][@ui-sref='content-view.versions']")
```

Comment:
does it have `Cancel` text? it will be better then

Reply:
really? why? i'd prefer unique attr over localized string

Reply:
because none of these attributes point on `cancel` specifically, if we have`[@ui-sref='content-view.versions.cancel']` your approach would be better

Reply:
just was my opinion, it is not better, but more logical